src/main/java/org/jenkinsci/plugins/verifystatus/VerificationsPublisher.java

```diff
@@ -100,65 +109,61 @@ public String getVerifyStatusRerun() {
     return verifyStatusRerun;
   }
 
+  public Short getVerifyStatusValue() {
```

Comment:
Maybe i'm just not understand the intention behind adding the verifyStatusValue.  I believe this was left out on purpose.  The UI does not have an input field for the value to prevent users from overriding the job result value.